asyncio_run_in_process/run_in_process.py

```diff
@@ -168,19 +175,41 @@ def open_in_process(
     async_fn: Callable[..., TReturn],
     *args: Any,
     loop: asyncio.AbstractEventLoop = None,
+    subprocess_kwargs: 'SubprocessKwargs' = None,
 ) -> AsyncContextManager[ProcessAPI[TReturn]]:
     return cast(
         AsyncContextManager[ProcessAPI[TReturn]],
-        _open_in_process(async_fn, *args, loop=loop),
+        _open_in_process(async_fn, *args, loop=loop, subprocess_kwargs=subprocess_kwargs),
     )
 
 
+def _update_subprocess_kwargs(subprocess_kwargs: Optional['SubprocessKwargs'],
+                              child_r: int,
+                              child_w: int) -> 'SubprocessKwargs':
+    if subprocess_kwargs is None:
+        subprocess_kwargs = {}
+
+    base_pass_fds = subprocess_kwargs.get('pass_fds', ())
+    pass_fds: Tuple[int, ...]
+
+    if base_pass_fds is None:
+        pass_fds = (child_r, child_w)
+    else:
+        pass_fds = tuple(set(base_pass_fds).union((child_r, child_w)))
```

Comment:
Can't this return a tuple with the items in the wrong order?

Reply:
I don't think order matters.  It's a set of file descriptors (integers) that will be kept open and the order shouldn't matter. I was just trying to deduplicate which may not be necessary but I don't see how it would cause problems...